scripts/irods/test/test_iadmin.py

```diff
@@ -1210,6 +1210,20 @@ def test_impostor_resource_debug_logging(self):
             server_config_update = {
                 'environment_variables': {
                     'spLogLevel': '11'
+                },
```

Comment:
we're not changing the logging in 4-2-stable... is this chunk necessary/appropriate?

Reply:
Good catch. I will remove this bit. Doesn't hurt to have it there as 4-2-stable doesn't care about it in the server_config.json so I won't run tests again unless anybody thinks it would be appropriate to do so.

Reply:
good to go without rerun.   this is a baseline for everything else that will get tested.

check the codacy